Main.py

```diff
@@ -0,0 +1,65 @@
+import sys
+import os
+import re
+from DataProcession import DataProcessing
+from WeatherData import WeatherData
+
+# Month names dictionary
+month = {
```

Comment:
Constants are always in UPPER CASE to keep it different from variable names. Also month is singular, it should months